docs/hyperdatasets/single_frames.md

```diff
@@ -218,11 +218,11 @@ There are also options to populate the instance with:
 * A dictionary of annotation objects - `annotations`
 * A URI link to a mask file for the frame - `mask_source`
 
-For more information, see [SingleFrame](google.com).
+For more information, see the SingleFrame reference.
```

Comment:
@pollfly MIssing code highlight.
"See the SingleFrame reference" is unclear. "See  `SingleFrame` class description"